lib/internal/process/pre_execution.js

```diff
@@ -589,7 +589,7 @@ function readPolicyFromDisk() {
     // no bare specifiers for now
     let manifestURL;
     if (require('path').isAbsolute(experimentalPolicy)) {
-      manifestURL = new URL(`file://${experimentalPolicy}`);
+      manifestURL = pathToFileURL(experimentalPolicy);
```

Comment:
Unrelated to percent-encoding bug but couldn't lines 587,590~597 be replaced with single
```mjs
const manifestURL = require('url').pathToFileURL(experimentalPolicy);
```
[`url.pathToFileURL()` uses `path.resolve()`](https://github.com/nodejs/node/blob/9fc57006c27564ed7f75eee090eca86786508f51/lib/internal/url.js#L1470) and [`path.resolve()` uses `process.cwd()`](https://github.com/nodejs/node/blob/9fc57006c27564ed7f75eee090eca86786508f51/lib/path.js#L178) for relative paths so the whole `else` detour looks redundant.

Reply:
I think the goal is to accept absolute URLs as well as paths (which I think is the only place in Node.js code that does that)

Reply:
Ah, makes sense for urlstring support.
Then with this change, it becomes a bit inconsistent because "absolute paths" are interpreted as paths while relative paths work as relative URLs, e.g.
```mjs
// --experimental-policy="left%20right"
new URL('left%20right', 'file:///path/to/cwd/').href === 'file:///path/to/cwd/left%20right'
// --experimental-policy="/path/to/cwd/left%20right"
url.pathToFileURL('/path/to/cwd/left%20right').href === 'file:///path/to/cwd/left%2520right'
```

Reply:
According to our tests, that was already the expected behavior